.github/workflows/build.yml

```diff
@@ -1186,6 +1186,16 @@ jobs:
         timeout-minutes: 5
         run: yarn test:assert
 
+      - name: Build E2E app with shimmed ESM globals
+        working-directory: dev-packages/e2e-tests/test-applications/${{ matrix.test-application }}
+        timeout-minutes: 5
+        run: yarn ${{ matrix.build-command || 'test:build:shimmed' }}
```

Comment:
do we need to make this a separate job? Can we just make this part of the build command run?

Reply:
We dont, that was a bit premature from me. I changed it to just run these from inside `build` and `test`, good call!